web.py

```diff
@@ -168,16 +168,34 @@ def get_pokemarkers():
     session.close()
 
     for pokemon in pokemons:
-        markers.append({
-            'id': 'pokemon-{}'.format(pokemon.id),
-            'type': 'pokemon',
-            'trash': pokemon.pokemon_id in config.TRASH_IDS,
-            'name': POKEMON_NAMES[pokemon.pokemon_id],
-            'pokemon_id': pokemon.pokemon_id,
-            'lat': pokemon.lat,
-            'lon': pokemon.lon,
-            'expires_at': pokemon.expire_timestamp,
-        })
+            content = {
+                'id': 'pokemon-{}'.format(pokemon.id),
+                'type': 'pokemon',
+                'trash': pokemon.pokemon_id in config.TRASH_IDS,
+                'name': POKEMON_NAMES[pokemon.pokemon_id],
+                'pokemon_id': pokemon.pokemon_id,
+                'lat': pokemon.lat,
+                'lon': pokemon.lon,
+                'expires_at': pokemon.expire_timestamp,
+            }
+            
+            try:
+                if pokemon.atk_iv is not None:
+                    iv = {
+                        'atk': pokemon.atk_iv,
```

Comment:
`content['iv'] = pokemon.atk_iv`
and as we have the try-except block the `if pokemon.atk_iv is not None:` is not needed
or @Noctem what do you say to this?